repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -574,6 +576,17 @@ export interface MyFaresPointToPointProduct {
     endDate: string;
 }
 
+export interface MyFaresOtherFaresProduct {
```

Comment:
Is this different to the one above?

Reply:
Yeah, one is what comes back from the product table, one is the fleshed out one using S3 matching JSON stuff.